src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIAssistantChatClient.cs

```diff
@@ -290,7 +311,6 @@ void IDisposable.Dispose()
                         runOptions.ToolConstraint = ToolConstraint.None;
                         break;
 
-                    case null:
```

Comment:
Is this related to the core change?

Reply:
Tangentially. I was validating what other areas we were overwriting a default with a different default. If this is null, then the user hasn't explicitly chosen a setting, so we should just leave it to whatever the service's default is.